lago/providers/libvirt/network.py

```diff
@@ -288,9 +291,8 @@ def make_ipv4(last):
                 net_xml.append(dns)
 
         LOGGER.debug(
-            'Generated Network XML\n {0}'.format(
-                ET.tostring(net_xml, pretty_print=True)
-            )
+            'Generated Network XML\n {0}'.
+            format(ET.tostring(net_xml, pretty_print=True))
```

Comment:
this change probably does not meet the styling rules

Reply:
That's yapf doing, not mine... Strange.

Reply:
Reverted.